milestone-delivery/deliveries/SaintsUndone-M2.md

```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../).**  
+
+* **PR Link:** [Saints Undone Grant Application PR](https://github.com/vechain/grant-program/pull/185). 
+* **Milestone Number:** 2
```

Comment:
I think this should be milestone 1 based on your [application](https://github.com/vechain/grant-program/blob/master/applications/SaintsUndone.md)

Reply:
Hii @libotony 

We have received the payments once and I thought this time it is gonna be the milestone delivery number 02 and hence is this PR.

Do you want me to change the application?

Thanks!

Reply:
Based on our rule, the payment will be immediately sent out after your application is accepted, so based on the content in the application and the process, I would suggest stick to milestone 1.